qa/rolling-upgrade/src/test/resources/rest-api-spec/test/old_cluster/10_basic.yaml

```diff
@@ -119,13 +169,17 @@
         body:
           query:
             match:
-              f1: "{{f1}}"
+              s1: "{{s1}}"
```

Comment:
Now that I think about it, this *looks* like it wouldn't work because the template is used in the mixed cluster tests. Can you verify that this works? Something like `gradle qa:rolling-upgrade:check` out to tell you.